libs/features/mas/src/mas-commerce-service.js

```diff
@@ -23,16 +23,21 @@ const MARK_READY = 'mas:ready';
  */
 export class MasCommerceService extends HTMLElement {
     static instance;
-    promise = null;
+    readyPromise = null;
+
+    static get observedAttributes() {
```

Comment:
I don't know if we need this.
I tried updates these attributes, and the price in the page hasn't changed.
I find risky or complicated  observing the attributes of `mas-commerce-service`.


![image](https://github.com/user-attachments/assets/6716bf7b-9dae-4dc6-8b9d-09d48c62e9da)